src/main/java/frc/robot/commands/Drive.java

```diff
@@ -16,12 +17,17 @@
 import frc.robot.subsystems.DrivebaseSubsystem;
 
 public class Drive extends CommandBase {
+  private static final double DEFAULT_MAX_ACCEL = 5.0;
```

Comment:
Shouldn't this be in the constants file?

Reply:
Yes, actually.